userguide/tutorials/aws.adoc

```diff
@@ -29,28 +33,79 @@ With the click of a button, the template will install and configure:
+
+An example of a valid routing table for would be 
+
+|===
+|Destination |Target
+
+|192.168.0.0/16
+|local
+
+|0.0.0.0/0
+|igw-08bbaa2909129b953
+|===
+
+
+Regarding the subnets, the following configuration would work:
+
+* Kill Bill server and KAUI server instances subnets: `192.168.64.0/18` (zone `1e`) and `192.168.192.0/18` (zone `1b`)
+* RDS subnets: `192.168.128.0/18` (zone `1e`) and `192.168.0.0/18` (zone `1b`)
+
+
+=== CloudWatch Integration
+
+Our AMIs come pre-configured with CloudWatch integration to allow for better diagnostics. Below is a screenshot of the metrics that come for free:
+
+
+image:/userguide/assets/img/aws/cloudwatch.png[align=center]
+
+
+In order to have business metrics, we also allow to install additional plugins suchs as our free https://github.com/killbill/killbill-analytics-plugin[Analytics plugin]. This provides a subscription billing management solution as feature-rich as popular SaaS platforms, but that you can control. Below is a screenshot of such business metrics:
 
 
-image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/aws/analytics.png[align=center]
+image:/userguide/assets/img/aws/analytics.png[align=center]
 
 
 For installation support along the way, reach out to `support@killbill.io`.
 
-=== Configuration options
+=== Configuration Options
 
 The installation supports the following configuration options:
 
 * *VpcId:* the VPC to use for the installation. In your AWS Console, go to *Services* and search for *VPC*. Under *Your VPCs*, locate the VPC ID you would like to use or create a new one.
-* *Subnets:* the subnets to use, associated with at least two different availability zones. In the VPC Dashboard, go to *Subnets* and find two subnets in your VPC *in two different availability zones*. Alternatively, create new ones (use 10.0.0.0/24 and 10.0.1.0/24 as the IPv4 CIDR for instance).
+* *Subnets:* the subnets to use for the KB and KAUi instances. They should be associated with at least two different availability zones. In the VPC Dashboard, go to *Subnets* and find two subnets in your VPC *in two different availability zones*. Alternatively, create new ones (use 10.0.0.0/24 and 10.0.1.0/24 as the IPv4 CIDR for instance).
+* *RDSSubnets:* the subnets to use for the RDS instance. The RDS instances require a subnet group with at least two different availability zones.
 * *KeyName:* name of an existing EC2 KeyPair to enable SSH access to the instances. You can create a new one by going to *Key Pairs* in your EC2 Dashboard.
 * *HTTPLocation:* IP address range allowed to access the load balancer (you can always use 0.0.0.0/0 initially and adjust access later on).
 * *EnvType:* environment purpose (test, prod, etc.)
 * *InstanceType:* the EC2 instance type to use for Kill Bill. Here are some guidelines:
-** `m5.large`: test environment
-** `c5.xlarge`: up to 5,000 subscriptions
+** `t2.medium` -> `t2.xlarge`: test environment
```

Comment:
Should we remove these guidelines? IIRC they were a bit arbitrary and we have no incentive to push people to use larger instances (same pricing).